### PR TITLE
Companies supporting jhipster

### PR DESCRIPTION
**Pull request by pascalgrimaud**
As discussed here: https://groups.google.com/forum/#!topic/jhipster-dev/PehjjCEdUQQ

I added a new logos folder as I couldn't use the one from  `companies using JHipster` : the quality wasn't good enough there because it's smaller

About consulting and training, I deleted the link in the menu

@deepu105 @jdubois : I'll let you review when you're back from vacations. Now, it's my turn for holidays :-)

### CODE REVIEW

**Comment by avdev4j** [comment]
LGTM!

Good job @pascalgrimaud

File: _layouts/default.html [hunk 1]
```diff
@@ -335,8 +335,6 @@
                                         <ul class="nav navbar-nav">
                                             <li {% if '/help/' == page.url %}class="active" {% endif %}><a href="{{ site.url }}/help/"><i class="fa fa-fw fa-users"></i> Community help</a></li>
                                             <li {% if '/bug-bounties/' == page.url %}class="active" {% endif %}><a href="{{ site.url }}/bug-bounties/"><i class="fa fa-fw fa-usd"></i> Bug bounties</a></li>
-                                            <li {% if '/consulting/' == page.url %}class="active" {% endif %}><a href="{{ site.url }}/consulting/"><i class="fa fa-fw fa-user-md"></i> Professional help</a></li>
```
Comment (murdos) [inline _layouts/default.html]: `consulting.md` and `training.md` should probably be removed, since nothing links to them anymore

Comment (jdubois) [reply _layouts/default.html]: Indeed! I remember removing pages causes some warnings in Google Webmaster tools, but nevermind, it's just not good to have old orphaned pages

Comment (pascalgrimaud) [reply _layouts/default.html]: Ok, I'll remove these 2 pages

**Comment by pascalgrimaud** [comment]
No, I don't ask them, so I'll remove Google (if I don't have answer from @saturnism) and Jetbrains.
About Xebia Labs, what do you think @deepu105 and @sendilkumarn ? Should I keep Xebia Labs ?
About Okta, I think I can keep it too (cc @mraible), there will be 2 logos for Okta, one as gold sponsor, one as supporting JHipster

**Comment by deepu105** [comment]
Yes you can keep XebiaLabs. Btw Sendil doesn't work at XebiaLabs anymore

Thanks & Regards,
Deepu

**Comment by jdubois** [comment]
@sendilkumarn oh congrats for your new job, when is it official?

**Comment by pascalgrimaud** [comment]
I'm merging this, as it looks OK

**Comment by sendilkumarn** [comment]
@jdubois It is official :)

**Comment by jdubois** [comment]
@sendilkumarn congratulation tweet sent :-)

**Pull request merged**